scripts/dts/extract_dts_includes.py

```diff
@@ -316,12 +316,6 @@ def extract_property(node_compat, yaml, node_address, prop, prop_val, names):
 
     node = reduced[node_address]
     label_override = None
```

Comment:
You could clean up the 'label_override' here and lines 373/374 also I guess

Reply:
Fixed